### DISCUSSION TITLE
Add additional objects to Copy-Dba* by 1.5

### DISCUSSION THREAD

**Discussion by potatoqualitee** [dataplat/dbatools#7519] [Ideas]
### Is this a *feature* OR *bug*:

feature

### System Details

At a later date, from spaghettidba's list - not maintenance plans tho cuz they were too complex I believe.

![image](https://cloud.githubusercontent.com/assets/8278033/23566637/1d9c8e6c-0053-11e7-9e72-84828b3ff169.png)

**Comment by wsmelton** [comment]
I would exclude DTS packages and replication from this list. (We do not have replication commands as it is to address it.)

What is `Registry Configuration` anyway?

Trying to address `cryptographic providers` will be a bit of a challenge as well since that deals with the EKM.

**Comment by wsmelton** [comment]
Pretty sure most of these are covered that we can with SMO. Copy command wise I'm not aware that many have asked for the objects we don't include, so far. Those are better tracked by creating issues for them individually.

**Comment by potatoqualitee** [comment]
I can convert this to a discussion, but I want to keep it all in one place at least for the reminder.

**Comment by cstratton144** [comment]
I really need copy mainenance plans please! they are not associated with ssis packages.

**Reply by wsmelton** [reply]
We are happy to take contributions if someone wants to take a crack at it.

We looked at it little over a year ago and could not get the command to the quality and consistency we wanted. If I recall it was when we got to moving plans between versions it gets very dirty.
